service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/create/azure/resourcemanager/storage/StorageManagerOperation.java

```diff
@@ -0,0 +1,8 @@
+package bio.terra.workspace.service.resource.controlled.flight.create.azure.resourcemanager.storage;
+
+import bio.terra.cloudres.common.CloudOperation;
+
+/** {@link CloudOperation} for using StorageManager API. */
+public enum StorageManagerOperation implements CloudOperation {
```

Comment:
Hm, `StorageManagerOperation`, `BaseStorageRequestData`, `CreateStorageAccountRequestData` should move into CRL, right?

I guess am OK with leaving them here for now -- but can we make a ticket? (or at least a TODO?)